### PR TITLE
improve(serverless): refactor scripts into their own package & run in CI

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

Previously the serverless scripts were stored in the core scripts directory. This PR refractors this to a separate package.

issue: #2164

**Summary**

All server less scripts and tests for the hub and spoke are moved to their own package.

**Details**

Note: this PR will break production as configs in gcp will need to be updated in accordance with this release.

**Issue(s)**

close: #2164

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -31,7 +31,6 @@
     "moment": "^2.24.0",
     "node-fetch": "^2.3.0",
     "solidity-coverage": "^0.7.2",
-    "supertest": "^4.0.2",
```
Comment (nicholaspai) [inline packages/core/package.json:34]: What was this?

**Review by nicholaspai — APPROVED** [review]

**Review by drumenbassik — APPROVED** [review]

File: packages/core/package.json (hunk 1 above)
Comment (chrismaree) [reply packages/core/package.json:34]: This was the package I was using to test serverless. as this serverless is now in it's own package this is no longer needed in core :)

**Review by mrice32 — APPROVED** [review]
Looks great! Do you need to add a truffle config to this directory?

**Comment by chrismaree** [comment]
> Looks great! Do you need to add a truffle config to this directory?

I didn't think so but looking at how we've configured other packages that re-use the common truffle config we should

File: packages/serverless-orchestration/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@uma/serverless-orchestration",
+  "version": "1.1.0",
+  "description": "Serverless orchestration scripts to run a parallel instances of scripts or bots",
+  "dependencies": {
+    "@truffle/contract": "^4.2.20",
+    "@uma/common": "^1.0.1",
+    "@uma/financial-templates-lib": "^1.1.0"
+  },
+  "devDependencies": {
```
Comment (mrice32) [inline packages/serverless-orchestration/package.json:10]: nit: add `@uma/core` as a dev dependency for the tests?

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**